specifyweb/frontend/js_src/lib/queryfield.js

```diff
@@ -11,6 +11,10 @@ var QueryFieldSpec    = require('./queryfieldspec.js');
 var QueryFieldInputUI = require('./queryfieldinput.js');
 const queryText = require('./localization/query').default;
 const commonText = require('./localization/common').default;
+const dataModelStorage = require('./wbplanviewmodel').default;
```

Comment:
I'd rather you not use Workbench code here. 

Use code similar to https://github.com/specify/specify7/blob/e369d1f81a4234d74998ca5c852f37c66d4869e0/specifyweb/frontend/js_src/lib/queryfield.js#L232-L245

Reply:
> Some day the tree definitions should be set up to just load at start up. But that is a broader scope fix than this issue.

dataModelStorage serves exactly that purpose. It stores the names and titles of all tree ranks

> I'd rather you not use Workbench code here.

wbplanviewmodel does not depend on other workbench files. It might be appropriate to rename it. Also, it is already used outside of the workbench (Leaflet and S^N)

And unlike schema.models, dataModelStorage has type definitions

Do you still want me to revert to fetching tree definitions here?

Reply:
Yes, please. The whole data model is not needed in this context, just the tree definition.